### PR TITLE
add workspace members empty page

### PR DESCRIPTION
**Pull request by StuartHarris**
[Ticket AB 2072](https://dev.azure.com/futurenhs/FutureNHS/_boards/board/t/FutureNHS%20Team/Stories/?workitem=2072)

## Acceptance Criteria

- [x] link in side bar
- [x] empty page according to design

## Pre-review checklist:

- [x] Tests

- [ ] Documentation

- [ ] Analytics (user analytics, e.g. Google Analytics)

- [ ] Observability (metrics/tracing)

- [ ] Feature flag

- [ ] Data columns adhere to the [Dublin Core Metatdata schema](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/). Further information is available [here](https://www.gov.uk/government/publications/recommended-open-standards-for-government/using-metadata-to-describe-data-shared-within-government)

## Testing information

- Is there any special setup required?

- Test plan?

## Test:

- [x] Tested locally

- Platforms/Browsers:

  - [ ] Google Chrome (latest version)

  - [ ] Internet Explorer 11

  - [ ] Edge

- Accessibility:

  - [ ] Screen Reader

  - [ ] Keyboard Navigation

  - [ ] Magnification

  - [ ] Contrast

  - [ ] Text size 200%

  - [ ] Touch target areas (Mobile)

  - [ ] Landscape (Mobile)

### CODE REVIEW

**Review by frigus02 — APPROVED** [review]
Looks good. Do you think the test should be moved from `frontend/__tests__/pages/workspaces/index.test.tsx` to `frontend/__tests__/pages/workspaces/[workspaceId]/index.test.tsx`?

**Comment by StuartHarris** [comment]
> Looks good. Do you think the test should be moved from `frontend/__tests__/pages/workspaces/index.test.tsx` to `frontend/__tests__/pages/workspaces/[workspaceId]/index.test.tsx`?

Wow, good spot! Thanks

File: frontend/pages/workspaces/[workspaceId]/members.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import { Navigation } from "../../../components/Navigation";
+import { PageLayout } from "../../../components/PageLayout";
+import { useGetWorkspaceByIdQuery } from "../../../lib/generated/graphql";
+import withUrqlClient from "../../../lib/withUrqlClient";
+
+const PageContent = styled.section`
+  flex-grow: 3;
+  min-height: 100vh;
+  padding-top: 24px;
+  padding-left: 10%;
```
Comment (declanslevin) [inline frontend/pages/workspaces/[workspaceId]/members.tsx:20]: I've noticed an inconsistency here copy-pasting this `PageContent` component across pages where on some we use %s, others we use px values. 

Is there a plan to standardise this into a shared component?

File: frontend/pages/workspaces/[workspaceId]/members.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+  return (
+    <>
+      <Head title={workspaceTitle} />
+      <PageLayout>
+        <NavHeader />
+        <ContentWrapper>
+          <Navigation workspaceId={id} workspaceTitle={workspaceTitle} />
+          <PageContent>
+            <MainHeading withBorder>Workspace members</MainHeading>
```
Comment (declanslevin) [inline frontend/pages/workspaces/[workspaceId]/members.tsx:50]: The `withBorder` prop was removed as part of the design sweep last week

**Review by declanslevin — COMMENTED** [review]
Pair reviewed with David

**Comment by declanslevin** [comment]
Well that was bad timing

**Comment by StuartHarris** [comment]
Yeah my bad, sorry

**Comment by StuartHarris** [comment]
Fixing now, in another PR

File: frontend/pages/workspaces/[workspaceId]/members.tsx (hunk 1 above)
Comment (StuartHarris) [reply frontend/pages/workspaces/[workspaceId]/members.tsx:20]: I think we should maybe move to a layout?

**Pull request merged**